From 13a531dde8ee181ad9e8aeb6b755ca52f261bef8 Mon Sep 17 00:00:00 2001
From: jakub <jakub@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Fri, 15 Apr 2011 10:29:58 +0000
Subject: [PATCH 123/200] 	PR target/48614
 	* gcc.target/i386/pr46084.c: Require avx_runtime instead of
 	just avx.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@172485 138bc75d-0d04-0410-961f-82ee72b054a4

index 88bcd1c..3ca22eb 100644
--- a/gcc/testsuite/gcc.target/i386/pr46084.c
+++ b/gcc/testsuite/gcc.target/i386/pr46084.c
@@ -1,7 +1,7 @@
 /* This test needs to use setrlimit to set the stack size, so it can
    only run on Unix.  */
 /* { dg-do run { target *-*-linux* *-*-solaris* *-*-darwin* } } */
-/* { dg-require-effective-target avx } */
+/* { dg-require-effective-target avx_runtime } */
 /* { dg-require-effective-target split_stack } */
 /* { dg-options "-fsplit-stack -O2 -mavx" } */
 
-- 
1.7.0.4

